### PR TITLE
Add edit/category extend manage categories template

### LINKED ISSUE #674: Add edit/category should extend manage categories template

### PR DESCRIPTION
**Pull request by AdriSal**
Fix #674

### CODE REVIEW

File: wouso/interface/apps/files/templates/files/cpanel/add_file.html [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{% extends 'files/cpanel/list_files.html' %}
+{% extends 'files/cpanel/manage_categories.html' %}
```
Comment (gabrielivascu) [inline wouso/interface/apps/files/templates/files/cpanel/add_file.html]: <code>add_file</code> template must extend <code>list_files</code> template. Please revert these changes here.

**Comment by iulianR** [comment]
LGTM :+1:
